airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -503,10 +503,17 @@
   icon: exchangeratesapi.svg
   sourceType: api
   releaseStage: alpha
+- name: Everhour
+  sourceDefinitionId: 6babfc42-c734-4ef6-a817-6eca15f0f9b7
+  dockerRepository: airbyte/everhour
+  dockerImageTag: 0.1.0
+  documentationUrl: https://docs.airbyte.com/integrations/sources/everhour
+  icon: everhour.svg
+  sourceType: api
+  releaseStage: alpha
   allowedHosts:
     hosts:
-      - "${subdomain}.apilayer.com"
-      - apilayer.com
```

Comment:
Were these removed intentionally? This hosts block was originally under the Exchange Rates API connector definition, above Everhour.

Reply:
Oh, no. @sh4sh I reverted the change.